themes/default/templates/macros.mtt

```diff
@@ -101,9 +101,9 @@
 		$$printTypeParams(::field.params::) (
 		::foreach arg args[0]::
 			<span style="white-space:nowrap">
-			::if arg.opt && arg.value == null::?::end::
-			::arg.name:::$$printLinkedType(::arg.t::)
-			::if arg.value != null && arg.value != "null":: = ::arg.value::::end::
+				::if arg.opt && (arg.value == null || arg.value == "null")::?::end::
```

Comment:
Hmm, that seems rather strange. Why is the value being null a special case? I don't quite see how that works for the `?b = "foo"` case.